src/confluent_kafka/src/confluent_kafka.h

```diff
@@ -42,28 +42,28 @@
  *  0xMMmmRRPP
  *  MM=major, mm=minor, RR=revision, PP=patchlevel (not used)
  */
-#define CFL_VERSION     0x02090000
-#define CFL_VERSION_STR "2.9.0"
+#define CFL_VERSION     0x02100000
+#define CFL_VERSION_STR "2.10.0rc3"
 
 /**
  * Minimum required librdkafka version. This is checked both during
  * build-time (just below) and runtime (see confluent_kafka.c).
  * Make sure to keep the MIN_RD_KAFKA_VERSION, MIN_VER_ERRSTR and #error
  * defines and strings in sync.
  */
-#define MIN_RD_KAFKA_VERSION 0x020800ff
+#define MIN_RD_KAFKA_VERSION 0x021000ff
```

Comment:
This is in hex, same for `CFL_VERSION`
```suggestion
#define MIN_RD_KAFKA_VERSION 0x020a00ff
```